conf/pipeline/sarek/azurebatch_pools_Ddv4.config

```
@@ -0,0 +1,61 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    nf-core/sarek Nextflow config file for Azure Batch pools
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines several Azure Batch pools with virtual machines from Ddv4-series, 
+    assigns pools to labels according to the requirements defined in base.config.
+    You might need to adjust vmCount and maxVmCount depending on your Batch account quotas.
+
+    Use as follows:
+        nextflow run nf-core/sarek -c '<Path to this configuration file>'  -profile azurebatch
```

```suggestion
        nextflow run nf-core/sarek -profile azurebatch
```

Using ` -profile azurebatch` should be enough since you included it in the profile

---

Done